CHANGELOG.md

```diff
@@ -2,53 +2,52 @@
 
 All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org).
 
+## [v1.1.0](https://github.com/puppetlabs/puppetlabs-resource_api/tree/v1.1.0) (2019-06-17)
+
+[Full Changelog](https://github.com/puppetlabs/puppetlabs-resource_api/compare/1.0.0...v1.1.0)
+
+### Added
+
+- PDK sync and slack notifications [\#18](https://github.com/puppetlabs/puppetlabs-resource_api/pull/18) ([DavidS](https://github.com/DavidS))
```

Comment:
Not a uservisible change. this should not be an `enhancement`, but rather maintenance.

Reply:
Will update now :-)